app/jobs/finalize_import_job.rb

```diff
@@ -0,0 +1,37 @@
+class FinalizeImportJob < ActiveJob::Base
```

Comment:
Ugh, I always make this mistake as the Vim plugin creates the file and inherits from `ActiveJob::Base` by default. Thanks for catching this! ❤️